.sobelow-skips

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
What does this file for? 🤔

Reply:
it's the result of running the` mix sobelow --mark-skip-all` command.
After running `sobelow` scan, there could be a number of false positives. We can skip those false positives by commenting above the code, e.g.
```elixir
# sobelow_skip ["Traversal"]
def vuln_func(...) do
  ...
end
```

but the comments don't work on template `.eex` file. In my case, the fault positive is on the use of `raw` method
https://github.com/longnd/elixir-gscraper/blob/dcdd8cf7cb24a059f4bc4cf609633c7526df361c/lib/gscraper_web/templates/shared/_header.html.eex#L12-L14

so the way to fix is to let `sobelow` skip those files by running the above command.

I found that solution here: https://github.com/nccgroup/sobelow/issues/84#issuecomment-784009058